vmdb/app/controllers/ops_controller.rb

```diff
@@ -680,7 +681,7 @@ def replace_right_cell(nodetype, replace_trees = false) # replace_trees can be a
             page.replace("ops_tabs", :partial=>"all_tabs")
           elsif nodetype == "log_depot_edit"
             @right_cell_text = "Editing Log Depot settings"
-            page.replace_html("diagnostics_collect_logs", :partial=>"layouts/edit_log_depot_settings")
```

Comment:
@chessbyte I verified that the log collection was the last piece using this partial, which means it can now be removed in this PR.
In the last update that I pushed, I have removed this partial. Thanks for the suggestion.